demos/apps/echo-with-auth/README.md

```diff
@@ -0,0 +1,55 @@
+# Istio Kubernetes Demo with simple echo app
+
+**Install Istio on a Kubernetes cluster**
+
+1. Install Istio Manager
+    ```bash
+    kubectl apply -f istio-manager.yaml
+    ```
+
+    Notice in the YAML file, the manager has been configured to use mutual TLS connections between Istio proxies
+    ```yaml
+    data:
+      mesh: |-
+        authPolicy: 1                        # Enables Istio auth
```

Comment:
Definitely, let's organize this the right way up front and not create debt for ourselves.

Please see this doc and other docs on istio.io to learn how to add stuff to the web site: https://istio.io/docs/home/contribute/writing-a-new-topic.html